docs/contributor-guide/developer-guide/deploy-system/developer-guide.md

```diff
@@ -0,0 +1,171 @@
+# Deploy system developer guide
+
+This page contains information for deploy system developers, which go beyond the initial setup. 
+You should have worked through the [Prerequisits & dev setup](contributor-guide/developer-guide/deploy-system/prerequisits.md) already
+and know the difference between the repositories (civiform-deploy, civiform-deploy-infra and civiform) from the overview section.
+
+The "Most common developer flows" section contains all essential information to be able to do a first change. 
+We recommend coming back soon to "Optimizing your flow" and skim reading the tips and tricks section so you know 
+what information is provided when you need it.
+
+## Running locally
+
+The most common flow for developers is to run the deployment script on your local machine to test your changes to 
+the civiform-deploy-infra repository. To do this,you create a configuration file for your deployment and then run 
```

Comment:
I wonder if this should actually be in prerequisites. Or maybe there's another way to more clearly delineate between the one time setup work we need to do (setting up AWS account, creating config file, etc.), and the steps for ongoing development.

Reply:
gotcha, thank you for explaining!